### PR TITLE
[CIVP-19310] Expose more new params

### PR DESCRIPTION
**Pull request by zacharydfriedlander**
Add new params from `civis_file_to_table` to `dataframe_to_civis` and `csv_to_civis`.

### CODE REVIEW

File: civis/io/_tables.py [hunk 1]
```diff
@@ -615,7 +615,9 @@ def dataframe_to_civis(df, database, table, api_key=None, client=None,
                        diststyle=None, distkey=None,
                        sortkey1=None, sortkey2=None,
                        headers=None, credential_id=None,
-                       polling_interval=None,
+                       primary_keys=None, last_modified_keys=None,
```
Comment (jkulzick) [inline civis/io/_tables.py:618]: We should add these to the docstring as well

File: civis/io/_tables.py [hunk 2]
```diff
@@ -730,8 +735,11 @@ def csv_to_civis(filename, database, table, api_key=None, client=None,
                  diststyle=None, distkey=None,
                  sortkey1=None, sortkey2=None,
                  delimiter=",", headers=None,
-                 credential_id=None, polling_interval=None,
-                 archive=False, hidden=True):
+                 primary_keys=None, last_modified_keys=None,
```
Comment (jkulzick) [inline civis/io/_tables.py:755]: We should add these to the docstring as well

**Review by jkulzick — CHANGES_REQUESTED** [review]

**Review by jkulzick — APPROVED** [review]
LGTM %Travis

**Comment by zacharydfriedlander** [comment]
So that other travis job has finished (and is green) but for some reason Github didn't get the memo here. Merging.

**Pull request merged**